src/signer.rs

```diff
@@ -6,10 +6,22 @@ use rustls::{
     sign::{Signer, SigningKey},
     Error, OtherError, SignatureAlgorithm, SignatureScheme,
 };
-use sha2::digest::Digest;
```

Comment:
sha2 crate isn't FIPS certified. The BCrypt APIs are FIPS certified. Since already depending on NCrypt APIs, change to use BCrypt hash APIs for FIPS story.